src/app/commands/bounty/Bounty.ts

```diff
@@ -206,6 +206,20 @@ export default class Bounty extends SlashCommand {
                     name: Activities.list,
                     type: CommandOptionType.SUB_COMMAND,
                     description: 'View list of bounties you created or are claimed',
+                    options: [
+                        {
+                           name: 'channel-category',
+                           description: 'Bounty Channel Category',
+                           type: CommandOptionType.STRING,
```

Comment:
Can we limit this to actual channel categories by using "options" with the list of existing categories?